cron/monitoring/exporter.go

```diff
@@ -16,22 +16,33 @@ package monitoring
 
 import (
 	"fmt"
+	"time"
 
 	"contrib.go.opencensus.io/exporter/stackdriver"
+	"contrib.go.opencensus.io/exporter/stackdriver/monitoredresource/gcp"
 
 	"github.com/ossf/scorecard/cron/config"
 )
 
-const stackdriverTimeSeriesQuota = 100
+const (
+	stackdriverTimeSeriesQuota = 200
+	timeout                    = "10m"
```

Comment:
Just do `10 * time.Minute` here. No need for the runtime parsing.

Reply:
Done. Thanks!